### PR TITLE
Create CCC_TAC_Minutes-2022-06-16.md

### PR DESCRIPTION
**Pull request by krtaylor**

### CODE REVIEW

File: TAC/Meetings/2022/2022-06-16/CCC_TAC_Minutes-2022-06-16.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+* Kurt Taylor (Linux Foundation)
+
+#### Not attending 
+* Nagaraju (Intel)
+* Megan Collier (Investment Group)
+* Nick Vidal (Profian)
+* Shalom Shefa (Cisco)
+* Thomas Hardjono (MIT)
+* Hannes Tschofenig (Arm)
+* Helen Lau (Linux Foundation)
```
Comment (dthaler) [inline TAC/Meetings/2022/2022-06-16/CCC_TAC_Minutes-2022-06-16.md:81]: What is the purpose of a "not attending" section?  Anyone on lines 42-50 without a `*` is also not attending....

Comment (dcmiddle) [reply TAC/Meetings/2022/2022-06-16/CCC_TAC_Minutes-2022-06-16.md:81]: yeah I'd discourage trying to track who isn't in attendance.

File: TAC/Meetings/2022/2022-06-16/CCC_TAC_Minutes-2022-06-16.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+* Mark Novak (JP Morgan Chase)
+* Simonn Johnson (Intel)
+* Suhas Hegde (Ruby Protocol)
+* Ionut Mihalcea (Arm)
+* Kurt Taylor (Linux Foundation)
+
+#### Not attending 
+* Nagaraju (Intel)
+* Megan Collier (Investment Group)
+* Nick Vidal (Profian)
```
Comment (krtaylor) [inline TAC/Meetings/2022/2022-06-16/CCC_TAC_Minutes-2022-06-16.md:77]:
```suggestion
```

File: TAC/Meetings/2022/2022-06-16/CCC_TAC_Minutes-2022-06-16.md [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+* Shalom Shefa (Cisco)
```
Comment (krtaylor) [inline TAC/Meetings/2022/2022-06-16/CCC_TAC_Minutes-2022-06-16.md:78]:
```suggestion
```

File: TAC/Meetings/2022/2022-06-16/CCC_TAC_Minutes-2022-06-16.md [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+* Thomas Hardjono (MIT)
```
Comment (krtaylor) [inline TAC/Meetings/2022/2022-06-16/CCC_TAC_Minutes-2022-06-16.md:79]:
```suggestion
```

File: TAC/Meetings/2022/2022-06-16/CCC_TAC_Minutes-2022-06-16.md [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+* Hannes Tschofenig (Arm)
```
Comment (krtaylor) [inline TAC/Meetings/2022/2022-06-16/CCC_TAC_Minutes-2022-06-16.md:80]:
```suggestion
```

File: TAC/Meetings/2022/2022-06-16/CCC_TAC_Minutes-2022-06-16.md (hunk 1 above)
Comment (krtaylor) [inline TAC/Meetings/2022/2022-06-16/CCC_TAC_Minutes-2022-06-16.md:81]:
```suggestion
```

**Pull request merged**